backend/src/main/java/com/woowacourse/levellog/feedback/dto/FeedbackContentDto.java

```diff
@@ -1,12 +1,17 @@
 package com.woowacourse.levellog.feedback.dto;
 
+import com.woowacourse.levellog.feedback.domain.Feedback;
+import com.woowacourse.levellog.levellog.domain.Levellog;
+import com.woowacourse.levellog.member.domain.Member;
 import javax.validation.constraints.NotNull;
 import lombok.AccessLevel;
 import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
 @Getter
+@EqualsAndHashCode
 @AllArgsConstructor
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class FeedbackContentDto {
```

Comment:
```suggestion
@Getter
@EqualsAndHashCode
@AllArgsConstructor(access = AccessLevel.PRIVATE)
@NoArgsConstructor(access = AccessLevel.PRIVATE)
public class FeedbackContentDto {
```
정적 팩토리 메서드를 도입했으니까 생성자는 PRIVATE으로 변경해주세요

Reply:
정적 팩토리 메소드는 엔티티에서 DTO로 변환할 때의 편리성을 위해 도입했어요!
from()만 남기고 AllArgs 생성자는 private으로 닫으면, 테스트 코드에서 Request 목적으로 DTO를 생성할 때 Feedback 도메인까지 생성해서 넣어줘야 해서 AllArgs 생성자는 public으로 열어놓는 게 좋을 거 같아요😀